src/components/library-app-5/navigations/routes.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import {NavigationContainer} from '@react-navigation/native';
+import {createNativeStackNavigator} from '@react-navigation/native-stack';
+
+import {BookForm, BookListing, ShowBook} from '../screens';
```

Comment:
Keep components and constants separate. 

Hierarchy should be
1. React & other libs
2. Components
3. Actions
4. Helpers
5. Constants
6. Assets